recipe-client-addon/bootstrap.js

```diff
@@ -29,7 +29,7 @@ const REASONS = {
 
 const PREF_BRANCH = "extensions.shield-recipe-client.";
 const DEFAULT_PREFS = {
-  api_url: "https://self-repair.mozilla.org/api/v1",
+  api_url: "https://normandy.services.mozilla.com/api/v1",
```

Comment:
Once we add an action for pref hotfixes, we should plan to send one out to update users (mostly Aurora/Nightly users) to the new URL who don't already have it as a default.